src/catalog/src/builtin.rs

```diff
@@ -6750,6 +6774,31 @@ ORDER BY 1, 2"#,
     access: vec![PUBLIC_SELECT],
 });
 
+pub static MZ_SHOW_CONTINUAL_TASKS: LazyLock<BuiltinView> = LazyLock::new(|| BuiltinView {
```

Comment:
Not required in this PR but we generally index the `SHOW X` commands, [example](https://github.com/MaterializeInc/materialize/blob/main/src/catalog/src/builtin.rs#L7297)